launchers/mdsp-connector/provider.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM gradle:7.4.1-jdk17 AS build
+
+COPY --chown=gradle:gradle . /home/gradle/project/
+WORKDIR /home/gradle/project/
+RUN gradle :samples:other:file-transfer-http-to-http:provider:shadowJar --no-daemon
+
+FROM openjdk:17-slim
+
+WORKDIR /app
+COPY --from=build /home/gradle/project/samples/other/file-transfer-http-to-http/provider/build/libs/provider.jar /app
+COPY --from=build /home/gradle/project/samples/other/file-transfer-http-to-http/provider/config.properties /app
```

Comment:
same as above (extra config)

Reply:
They are not necessarily secrets in the config.properties. just additional settings not set in the java executable call. The secrets are handled differently via the Helm Chart.